server/service/src/sync/translations/stocktake_line.rs

```diff
@@ -48,6 +48,11 @@ pub struct LegacyStocktakeLineRow {
     #[serde(rename = "optionID")]
     #[serde(deserialize_with = "empty_str_as_option_string")]
     pub inventory_adjustment_reason_id: Option<String>,
+
+    #[serde(rename = "om_item_variant_id")]
+    #[serde(deserialize_with = "empty_str_as_option_string")]
+    #[serde(default)]
+    pub item_variant_id: Option<String>,
```

Comment:
mSupply iss created to handle this field
https://github.com/msupply-foundation/msupply/issues/15618